case-lib/lib.sh

```diff
@@ -140,10 +140,10 @@ func_lib_start_log_collect()
         return 3
     fi
 
-    local loggerCmd="$SOFLOGGER $logopt -l $ldcFile -o $logfile"
-    dlogi "Starting $loggerCmd"
+    local loggerCmd=("$SOFLOGGER" "$logopt" -l "$ldcFile" -o "$logfile")
+    dlogi "Starting ${loggerCmd[*]}"
```

Comment:
loggerCmd is array? how? below sudo "${loggerCmd[@]}" also same question. Quoting is good

Reply:
`"${name[@]}"` cannot be used inside a larger string, I mean you cannot do `"blabla ${name[@]}"` because that does not make sense. `"${name[@]}"` expands to: `"element 1" "element 2" ...`  so where would `"blabla"` be in that list?